content/en/continuous_integration/tests/dotnet.md

```diff
@@ -45,12 +45,15 @@ To report test results to Datadog, you need to configure the Datadog .NET librar
 
 {{% tab "On-Premises CI provider (Datadog Agent)" %}}
 
-If you are running tests on an on-premises CI provider, such as Jenkins or self-managed GitLab CI, install the Datadog Agent on each worker node by following the [Agent installation instructions][1]. This is the recommended option as test results are then automatically linked to the underlying host metrics.
+If you are running tests on an on-premises CI provider, such as Jenkins or self-managed GitLab CI, install the Datadog Agent on each worker node by following the [Agent installation instructions][1]. This is the recommended option as test results are then automatically linked to the underlying host metrics. CI Visibility uses the **traces** endpoints in the Agent.
```

Comment:
```suggestion
If you are running tests on an on-premises CI provider, such as Jenkins or self-managed GitLab CI, install the Datadog Agent on each worker node by following the [Agent installation instructions][1]. This is the recommended option as test results are then automatically linked to the underlying host metrics.
```

I think it's more confusing than anything. As we are already asking users to set the `DD_TRACE_AGENT_URL` env var later on, we should be covered.